tests/unit/router.tap.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+var tap = require('tap')
+var utils = require('@newrelic/test-utilities')
+var methods = require('methods')
+var instrumentation = require('../../lib/router-instrumentation.js')
+
+var WRAPPED_METHODS = [
+  'param'
+]
+
+var UNWRAPPED_METHODS = methods.concat([
+  'register',
+  'use',
+  'prefix',
+  'routes',
+  'middleware',
+  'allowedMethods',
+  'all',
+  'redirect',
+  'route',
+  'url',
+  'match'
+])
+
+var UNWRAPPED_STATIC_METHODS = [
+  'url'
+]
+
+tap.test('koa-router', (t) => {
+  var helper = utils.TestAgent.makeInstrumented()
+  t.tearDown(() => helper.unload())
```

Comment:
I forget, will this only run on later versions of node? Either way, I'd say for consistency's sake swap out the arrows for `function`s (unfortunately).